src/orgasm_control.c

```diff
@@ -314,13 +318,10 @@ static void orgasm_control_updateEdgingTime() { // Edging+Orgasm timer
             eom_hal_set_encoder_rgb(0, 255, 0);
         }
 
-        // raise motor speed to max speep. protect not to go higher than max
-        if (output_state.motor_speed <= (Config.motor_max_speed - calculate_increment(
-                Config.motor_start_speed, Config.motor_max_speed, Config.motor_ramp_time_s))
+        // raise motor speed to max speed. protect not to go higher than max
```

Comment:
This scope here is a good place to put your `motor_increment` variable. You can declare it at the top of this scope, then define it down here with the calculation. That will leave it available to both the conditional and the speed assignment below.

Reply:
I refreshed my branch with your new updates since it was causing merge issues. Had to resubmit a merge request